src/core/core.js

```diff
@@ -135,6 +135,7 @@ export default class Core {
    * @param {boolean} query.append If true, adds the results of this query to the end of the current results, defaults false
    */
   verticalSearch (verticalKey, options = {}, query = {}) {
+    window.performance.mark('yext.answers.queryStart');
```

Comment:
Just do have the granularity, let's have separately named marks for vertical vs. universal search. That may tell us something interesting.